client/my-sites/sidebar/publish-menu.jsx

```diff
@@ -156,7 +166,7 @@ var PublishMenu = React.createClass( {
 				className={ className }
 				link={ link }
 				buttonLink={ menuItem.buttonLink }
-				onNavigate={ this.props.onNavigate }
+				onNavigate={ this.maybeRecordPostTypeStats }
```

Comment:
Since at this point we know the post type, I think we could simplify this to bind the stats tracking at the point of render:

```jsx
const isCustomPostType = ! includes( [ 'post', 'page' ], menuItem.name );
return <SidebarItem onNavigate={ isCustomPostType ? this.recordPostTypeStats : this.props.onNavigate } />
```

Would eliminate the need to pass around `className` from the item component.